pkg/configurations/configurations.go

```diff
@@ -0,0 +1,159 @@
+package configurations
+
+import (
+	"fmt"
+	"os"
+	"path/filepath"
+	"sync"
+
+	"github.com/gobwas/glob"
+
+	"github.com/fsnotify/fsnotify"
+
+	"github.com/thoas/go-funk"
+	logf "sigs.k8s.io/controller-runtime/pkg/runtime/log"
+
+	"github.com/spf13/viper"
+)
+
+const defaultInterfacesFilter = "veth*"
```

Comment:
We can get rid of it and handle it in the initialization of `config`. Just add a comment that we are initializing with default values.

Reply:
but why not to have defaults as consts?

Reply:
It's unnecessary duplication. Unless we want to reinitialize the config. Mm, maybe it's just a matter of style. I leave it to you, since I don't really have a convincing explanation.